src/ServiceControl/Infrastructure/MarkerFile.cs

```diff
@@ -12,6 +12,7 @@ public MarkerFileService(string rootPath)
 
         public IDisposable CreateMarker(string name)
         {
+            Directory.CreateDirectory(rootPath);
```

Comment:
We discussed this before that we cannot do this during runtime. The assumption has to be that the directory got created by the installer.

Reply:
Hmmm... but it breaks the `F5` experience :-(

Reply:
We can assume SC has write access to the log folder, what about creating the directory in there?

Reply:
hmmm that sounds good

Reply:
The directory IS the log folder and that's why we debated this at length in the past ;) and decided not to add that CreateDirectory. Or if we do for "debugging" purposes we have to properly guard it with try catch and potentially Directory.Exist

Reply:
👍 for try-catch